go.mod

```diff
@@ -18,14 +18,38 @@ require (
 )
 
 require (
+	github.com/chigopher/pathlib v0.12.0 // indirect
```

Comment:
What is the go version locally? Also you did not add any new dependency intentionally, right?

Reply:
Updated my go version to 1.19.